activesupport/lib/active_support/callbacks.rb

```diff
@@ -826,6 +754,30 @@ def get_callbacks(name)
       def set_callbacks(name, callbacks)
         send "_#{name}_callbacks=", callbacks
       end
+
+      def default_false_terminator
+        Proc.new do |target, result_lambda|
+          terminate = true
+          catch(:abort) do
+            result = result_lambda.call if result_lambda.is_a?(Proc)
+            if CallbackChain.halt_and_display_warning_on_return_false && result == false
```

Comment:
I'm :+1: on doing this. `CallbackChain` doesn't feel like something people should know about.

But @claudiob, doesn't having the setting on `ActiveSupport::Callbacks` feel out of place with you saying it only affects Active Record and Active Model callbacks?